src/OperationProcessor.ts

```diff
@@ -233,10 +240,22 @@ class OperationProcessorImpl implements OperationProcessor {
       throw Error('Invalid operation: batchFileHash undefined');
     }
 
+    this.operationStore.store(opHash, operation);
+
+    const did = this.getDid(operation, opHash);
+    this.getDeferredOperationsList(did).append(opHash);
+
+    return opHash;
+  }
+
+  /**
+   * Processes a specified DID state changing operation.
+   */
+  private async processDeferred (operation: WriteOperation, opHash: OperationHash): Promise<void> {
```

Comment:
Move this private method below all public methods (after `processDeferredOperationsOfDid`).